core/src/main/java/gov/nist/secauto/metaschema/core/metapath/cst/AbstractCSTVisitorBase.java

```diff
@@ -28,6 +28,24 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.Nullable;
 
+/**
+ * Provides utility methods for processing Metapath abstract syntax tree (AST)
+ * nodes to produce a compact syntax tree (CST).
+ * <p>
+ * This base class implements common visitor patterns for transforming AST nodes
+ * into a more compact representation. The CST is optimized for efficient
+ * evaluation of Metapath expressions.
+ * <p>
+ * Key utility methods include:
+ * <ul>
+ * <li>{@link #nairyToList} - Processes n-ary expressions into a list</li>
```

Comment:
Potentially blocking because of function name, not just docs: sorry to be "that person," but I noticed this before and avoided bringing it up out of context. We do mean "[n-ary](https://xlinux.nist.gov/dads/HTML/naryfunc.html)" not n-airy, right?

Do we want to fix this in docs and functions?